theia-ide/generator-theia-extension
# Refined and fixed label provider template

fixed #56 

Refines and fixes the label provider template

How to test:

1. clone the repository
2. perform yarn
3. perform npm link
4. create a directory and perform yo theia-extension
5. select the LabelProvider option, and verify the build
6. cd browser-app => yarn start
7. Open theia and create a file with .my extension
9. Check that this file has a "star" icon and the label is appended with "with my label"

Signed-off-by: Jonas Helming <jhelming@eclipsesource.com>
